arch/risc-v/src/bl602/bl602_os_hal.h

```diff
@@ -0,0 +1,144 @@
+/****************************************************************************
+ * arch/risc-v/src/bl602/bl602_os_hal.h
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.  The
+ * ASF licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ ****************************************************************************/
+
+#ifndef __ARCH_RISCV_SRC_BL602_OS_HAL_H
+#define __ARCH_RISCV_SRC_BL602_OS_HAL_H
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+#include <assert.h>
```

Comment:
bl602_os_hal.h  is shared by the binary library and the open source code, it's better to move all header file inclusion to .c except stdint.h